### PR TITLE
Run the installer with --upgrade for all installs

### LINKED ISSUE #183: Run migrations after clone
After a clone has been run, we should run migrations `foreman-rake db:migrate`. I had a case where the hostname change script failed because of pending migrations when I ran it right after a clone

### PR DESCRIPTION
**Pull request by johnpmitsch**
Fixes #183
I had to do this with satellite 6.2.9 after restoring a backup from an older version.

### CODE REVIEW

**Comment by johnpmitsch** [comment]
Dolly run:

[62_rhel7]
192.168.121.212

[61_rhel7]
192.168.121.194

[62_rhel6_migration]
192.168.121.161

[62_rhel6_migration_invalid]
192.168.121.146

[62_online_with_pulp_data]
192.168.121.28

[62_online_without_pulp_data]
192.168.121.142

192.168.121.142            : ok=19   changed=16   unreachable=0    failed=0
192.168.121.146            : ok=18   changed=15   unreachable=0    **failed=1** (expected bad backup failure)
192.168.121.161            : ok=18   changed=15   unreachable=0    **failed=1** (db:seed error on initial installation, I re-ran the playbook and it succeeded, pretty sure this is a transient installer error)
192.168.121.194            : ok=18   changed=15   unreachable=0    failed=0
192.168.121.212            : ok=19   changed=16   unreachable=0    failed=0
192.168.121.28             : ok=19   changed=16   unreachable=0    failed=0

File: roles/satellite-clone/tasks/main.yml [hunk 1]
```diff
@@ -120,6 +120,10 @@
 - name: Wait for foreman-tasks service to start
   pause: minutes=5
 
+- name: Run installer upgrade
+  command: satellite-installer --upgrade
+  when: satellite_version == 6.2
```
Comment (sthirugn) [inline roles/satellite-clone/tasks/main.yml]: I am curious why only 6.2 and not 6.1?

Comment (johnpmitsch) [inline roles/satellite-clone/tasks/main.yml]: @sthirugn I haven't had to do this step for 6.1, so I just made the change for 6.2

Comment (sthirugn) [inline roles/satellite-clone/tasks/main.yml]: ok can you add a note that this is a mandatory step for now for Migration, 6.2

Comment (sthirugn) [inline roles/satellite-clone/tasks/main.yml]: something like

- name: Run installer upgrade (Currently for Migration and 6.2 cloning only)

**Comment by sthirugn** [comment]
ACK

**Comment by sthirugn** [comment]
ty @johnpmitsch

**Pull request merged**